File: ohyeah/app/src/main/java/com/ovatsia/ohyeah/MainActivity.java
package com.ovatsia.ohyeah;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView tw;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tw = findViewById(R.id.ohboy);
        tw.setText("oispa kaljaa");
    }
}

File: Piirto/app/src/main/java/com/ovatsia/piirto/MainActivity.kt
package com.ovatsia.piirto

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        var mv : MunView = MunView(this, null)
        setContentView(R.layout.activity_main)
    }
}
